src/app/components/Input/InputField.tsx

```diff
@@ -113,17 +113,15 @@ const styles = {
     px-2
     pt-5
     text-[17px]
-    text-black dark:text-white
     leading-[24px]
-    bg-white dark:bg-black
+    bg-primaryBg-light dark:bg-primaryBg-dark
     rounded-[4px]
     border
-    border-[rgba(15,20,25,0.1)] dark:border-[#333639]
     outline-none
 
-    focus:border-[#1d9bf0] dark:focus:border-[#1d9bf0]
-    invalid:border-[rgba(255,20,25,0.1)] dark:invalid:border-[#ff3639]
-    valid:border-[rgba(15,20,25,0.1)] dark:valid:border-[#333639]
+    focus:border-logo-light dark:focus:border-logo-light
```

Comment:
Is there a need to specify the same color on dark theme?

Reply:
Yes, because it won't work without it, only in light theme. (checked)